### ISSUE TITLE
Using custom Image Icon 

### ISSUE THREAD

**Issue by vishnuc** [react-native-community/react-native-tab-view#360]
I want to use custom image icon rather than default icons

```
  state: State = {
    index: 0,
    routes: [
      { key: '1', title: 'Hot Deals', icon: 'ios-flame' },
      { key: '2', title: 'All Deals', icon: 'ios-basket' },
      { key: '3', title: 'Filter', icon: 'ios-search' },
    ],
  };
```

How can I do that ?

**Comment by tuandt** [comment]
Hi,

You can implement  _renderIcon = (props: any) => ({ route, index }) => {
<Icon
          name={route.icon}
          color={color}
        />
}

then pass it to TabBar:  renderIcon={this._renderIcon(props)}

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
